lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -1683,20 +1685,35 @@ public void realtime_presence_attach_implicit_subscribe_fail() throws AblyExcept
 		AblyRealtime ably = null;
```

Comment:
Right, I'm sorry, and I was thinking of that requirement :)

What I want is something that checks whether or not the listener was added, in the failure case. Then pass or fail according to what the spec says :)